application-mocca-calendar-ui/src/main/resources/MoccaCalendar/Code/Macro.xml

```diff
@@ -1437,6 +1526,55 @@ $xwiki.ssx.use("MoccaCalendar.Code.Macro")
     &lt;/div&gt;
   &lt;/div&gt;
 #end
+
+#macro(importCalendarFileModal)
+  &lt;div class="modal fade" id="importCalendarFile" tabindex="-1" role="dialog"&gt;
+    &lt;div class="modal-dialog"&gt;
+      &lt;div class="modal-content"&gt;
+        &lt;div class="modal-header"&gt;
+          &lt;button type="button" class="close" data-dismiss="modal"&gt;&amp;times;&lt;/button&gt;
+          &lt;h4 class="modal-title"&gt;
+            $escapetool.xml($services.localization.render('MoccaCalendar.import.modal.title'))
+          &lt;/h4&gt;
+        &lt;/div&gt;
+        &lt;div class="modal-body"&gt;
+          #set ($escapedCalendarSpace = $escapetool.xml($doc.getSpace()))
+          #set ($escapedCalendarName = $escapetool.xml($doc.getDocumentReference().getName()))
+          #set ($actionURL = "$request.getContextPath()/rest/moccacalendar/import")
+          &lt;form class="xform" id="importForm" action="$actionURL" method="post"&gt;
+            &lt;div class="import-form"&gt;
+              &lt;label for="importCalendarParent"&gt;
+                $escapetool.xml($services.localization.render('MoccaCalendar.calendar'))&lt;/label&gt;
+              &lt;select id="importCalendarParent" name="parentCalendar"&gt;
+                #foreach($item in $services.moccacalendar.getAllCalendars()) ## TODO: add filter here, see MOCCACAL-76
+                  #set($itemdoc = $xwiki.getDocument($item))
+                  #if($!{itemdoc} &amp;&amp; ${itemdoc.hasAccessLevel("edit")})
+                    #set($selected="")
+                    #if($itemdoc.getId() == $doc.getId())
+                      #set($selected=" selected='selected'")
+                    #end
+                    &lt;option value="$escapetool.html($itemdoc.getFullName())"$selected&gt;
+                      $itemdoc.getDisplayTitle()&lt;/option&gt;
+                  #end
+                #end
+              &lt;/select&gt;
+              &lt;label for="immportFileInput"&gt;$escapetool.xml($services.localization.render(
+                'MoccaCalendar.import.modal.file.label'))&lt;/label&gt;
+              &lt;input type="file" id="immportFileInput" accept=".ics" name="importedfile"
+                data-max-file-size="$!escapetool.xml($xwiki.getSpacePreference('upload_maxsize'))"&gt;
```

Comment:
Are you still using it?

Reply:
I am not sure what you are referring to, could you be more specific?

Reply:
I am asking about `data-max-file-size`

Reply:
Already fixed in a previous commit

Reply:
Fixed this in a previous commit: https://github.com/xwikisas/application-mocca-calendar/pull/116/commits/199747f6467f49e0df74128128ebc300cc912680#diff-4707390ed2b87d3f6730ea39b32c6b9024cc738d4be2a77d137023b3f5bc449dR928